wallet/src/cli.rs

```diff
@@ -10,6 +10,7 @@ use tuxedo_core::types::OutputRef;
 
 use crate::{h256_from_string, keystore::SHAWN_PUB_KEY, output_ref_from_string, DEFAULT_ENDPOINT};
 
+use crate::{money::DEFAULT_NUM_OF_COINS};
```

Comment:
Please move the const to this file and change its name to `DEFAULT_MINT_VALUE`.

Reply:
Yes, it's done.